engine/api/v2_project_vcs.go

```diff
@@ -202,6 +202,15 @@ func (api *API) getVCSProjectHandler() ([]service.RbacChecker, service.Handler)
 			if err != nil {
 				return err
 			}
+
+			vcsClear, err := vcs.LoadVCSByID(ctx, api.mustDB(), pKey, vcsProject.ID, gorpmapping.GetOptions.WithDecryption)
```

Comment:
why do you need to load it twice ?

Reply:
Need some data from the auth encrypted column.
I had 2 choices:
* load one and clear the correct columns
* load twice and copy the column i want
I choose the second one